src/Tickets/Commerce/Gateways/Stripe/Stripe_Elements.php

```diff
@@ -54,8 +54,15 @@ public function include_payment_element() {
 
 		$payment_methods = ( new Merchant() )->get_payment_method_types();
 
-		// Don't load the Payment Element if just the Credit Card method is selected.
-		return ! ( 1 === count( $payment_methods ) && 'card' === $payment_methods[0] );
+		if ( 1 < count( $payment_methods ) ) {
+			return true;
+		} elseif ( 1 === count( $payment_methods ) && 'card' !== $payment_methods[0] ) {
+			return true;
+		} elseif ( 1 === count( $payment_methods ) && 'card' === $payment_methods[0] && $this->wallets_enabled() ) {
+			return true;
+		}
+
+		return false;
```

Comment:
Instead of this - which creates an ETP dependency, let's add a filter and have ETP hook into it.

Something like:
```suggestion
		return (bool) apply_filters( 'tec_ticekts_include_payment_method', false );
```

Then kill off the `wallets_enabled` function - let ETP handle the logic (will require a linked ETP PR)

Reply:
Yes, makes more sense, will do, thanks!